jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/list/ProcessInstanceListViewImpl.java

```diff
@@ -421,8 +441,59 @@ public void onClick( ClickEvent event ) {
 
         controlBulkOperations();
     }
+    
+    private Column<ProcessInstanceSummary, ProcessInstanceSummary> initErrorCountColumn() {
+        
+        Column<ProcessInstanceSummary, ProcessInstanceSummary> column = new Column<ProcessInstanceSummary, ProcessInstanceSummary>(
+                new AbstractCell<ProcessInstanceSummary>(BrowserEvents.CLICK){
+
+            @Override
+            public void onBrowserEvent(Context context, Element elem, ProcessInstanceSummary val, NativeEvent event,
+                    ValueUpdater<ProcessInstanceSummary> valueUpdater) {
+                super.onBrowserEvent(context, elem, val, event, valueUpdater);
+                if(BrowserEvents.CLICK.equals(event.getType()) && val != null) {
+                    errCountPopup.setData(val.getProcessInstanceId(),  val.getErrorCount());
+                    errCountPopup.showAt(event.getClientX(), event.getClientY());
+                }
+            }
+
+            @Override
+            public void render(Context context, ProcessInstanceSummary value, SafeHtmlBuilder sb) {
+                if(value != null){
+                    sb.append(value.getErrorCount());
+                }                
+            }
+            
+        }){
+
+            @Override
+            public String getCellStyleNames(Context context, ProcessInstanceSummary pis) {
+                String styles = super.getCellStyleNames(context, pis);
+                if(pis != null){
+                    ProcessRuntimeCSS css = ProcessRuntimeResources.INSTANCE.css();
+                    styles = (styles == null ? "" : styles + " ") + css.processInstanceErrorCount();
```

Comment:
@rbarriuso I think in this case you don't customize the cell div but append a new element in there. Which I think would be a badge (http://www.patternfly.org/pattern-library/widgets/#badges) with different text/colors. Perhaps we can double check with the UX team.